src/test/integration/services/transaction.get.e2e-spec.ts

```diff
@@ -85,6 +84,9 @@ describe('Transaction Get Service', () => {
           throw new HttpException({ status: HttpStatus.BAD_GATEWAY, message: "BAD GATEWAY" }, HttpStatus.BAD_GATEWAY);
         }));
 
+      jest.spyOn(transactionGetService['logger'], 'error').mockImplementation(() =>
+        "Unexpected error when getting transaction from elastic, hash '51ffbf3d27e06fd509c510ef0ff1ea7329359dba89c05aeec333de52a405664d'");
```

Comment:
this should throw an error?

Reply:
This throw an error
```
jest
        .spyOn(IndexerService.prototype, "getTransaction")
        // eslint-disable-next-line require-await
        .mockImplementation(jest.fn(async (_: string) => {
          throw new HttpException({ status: HttpStatus.BAD_GATEWAY, message: "BAD GATEWAY" }, HttpStatus.BAD_GATEWAY);
        }));
```
        
The highlighted code from this comment, it refers to the fact that I also treat the error log that appears after the execution of the test